### PR TITLE
Add support for ISO version of Durgod Hades

### PR DESCRIPTION
**Pull request by ebastler**
This PR expands QMK support for the Durgod Hades board (based on the ANSI firmware provided by [J-Sully](https://github.com/J-Sully) and [dkjer](https://github.com/dkjer)) to the ISO version of this board. 

## Description
The existing ANSI version folder has been renamed, an ISO version added. Rules.mk paths have been adapted to ensure existing commands to build Durgod Hades will still build the ANSI version.

This firmware has been tested by a user on the QMK Discord and they confirmed both switches and LED positions to be working as intended.

I paid attention not to change anything else about the code, as it is shared between various Durgod PCBs and I lack hardware access to test anything but the ISO version of Hades which has been added with this PR.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by ebastler** [comment]
@dkjer @J-Sully since you provided the ANSI firmware, and adding the ISO version required me to make some changes to file paths of the ANSI version, I'd like to ping you here and ask if my changes are okay for you as well.

File: keyboards/durgod/hades/iso/rules.mk [hunk 1]
```diff
@@ -0,0 +1 @@
+DEFAULT_FOLDER=durgod/dgk6x/hades_iso
```
Comment (dkjer) [inline keyboards/durgod/hades/iso/rules.mk:1]: Recommend mirroring this in `durgod/hades/ansi/rules.mk`. 

I have created a PR for this: https://github.com/ebastler/qmk_firmware/pull/1

**Comment by dkjer** [comment]
This looks good to me!

File: keyboards/durgod/hades/iso/rules.mk (hunk 1 above)
Comment (ebastler) [reply keyboards/durgod/hades/iso/rules.mk:1]: Oh, out of curiosity - did you get any files from durgod, or did you actually manage to reverse engineer the whole PCB from scratch? The ISO version only needed few changes, so it was mostly trial and error for me (and I happened to find the right way at first attempt), but reversing a whole per-key RGB PCB sounds really tough.

File: keyboards/durgod/dgk6x/hades_iso/config.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+#pragma once
+
+
+/* key matrix rows */
+#define MATRIX_ROW_PINS { A0, A1, A2, A3, A4 }
+
+/* key matrix columns */
+#define MATRIX_COL_PINS { B0, B1, B2, B10, B11, B12, B13, B14, B15, C6, C7, C8, C9, C10, C11 }
```
Comment (drashna) [inline keyboards/durgod/dgk6x/hades_iso/config.h:24]: Should be moved to the info.json config.

Comment (ebastler) [reply keyboards/durgod/dgk6x/hades_iso/config.h]: Check - did the same for the ANSI version as well, since I was already at it :)

**Review by drashna — APPROVED** [review]

File: keyboards/durgod/hades/iso/rules.mk (hunk 1 above)
Comment (dkjer) [reply keyboards/durgod/hades/iso/rules.mk:1]: No files from durgod; the initial version for K320 was based off from kuenhlee.  The Hades/Venus/Galaxy board is similar to the K320 board, but required tweaking timings and tracking down wiring.

**Comment by tzarc** [comment]
Retargeted to `develop` due to moves.
Please resolve the conflicts and update the keyboard aliases file so that old invocations against `hades` now point at `hades_ansi`.

**Comment by ebastler** [comment]
Sorry for the ugly force-push, saw no other way to resolve conflicts than re-create my branch off develop and re-apply changes. Also moved to data driven, and used aliases instead of DEFAULT_FOLDER lines in rules.mk files.

**Review by tzarc — APPROVED** [review]

File: keyboards/durgod/dgk6x/hades_iso/config.h [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+#pragma once
+
+#ifdef RGB_MATRIX_ENABLE
+#define DRIVER_1_LED_TOTAL 59
+#define DRIVER_2_LED_TOTAL 10   
+
+#define CAPS_LED 29
+#endif
+
```
Comment (fauxpark) [inline keyboards/durgod/dgk6x/hades_iso/config.h:25]:
```suggestion
#define DRIVER_1_LED_TOTAL 59
#define DRIVER_2_LED_TOTAL 10   

#define CAPS_LED 29
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request merged**